pkg/apis/ocibuilder/v1alpha1/types.go

```diff
@@ -375,3 +372,9 @@ type Command struct {
 	// +listType=map
 	Value []string `json:"value" protobuf:"bytes,7,opt,name=value"`
 }
+
+// Represents build image metadata
+type ImageMeta struct {
```

Comment:
Unless it is an industry standard acronym, we should spell out variables. 

`ImageMeta`
to
`ImageMetadata`

Reply:
This came in from #39 - will make the change there.